lua/cmp_tabnine/source.lua

```diff
@@ -104,8 +111,8 @@ Source._do_complete = function(ctx)
 	local max_lines = conf:get('max_lines')
 	local cursor = ctx.context.cursor
 	local cur_line = ctx.context.cursor_line
-	local cur_line_before = string.sub(cur_line, 0, cursor.col)
-	local cur_line_after = string.sub(cur_line, cursor.col + 1) -- include current character
+	local cur_line_before = string.sub(cur_line, 1, cursor.col - 1)
+	local cur_line_after = string.sub(cur_line, cursor.col) -- include current character
```

Comment:
These are wrong, and do not send the proper data to tabnine.
consider the following code:
```python
var_a = 10                 
var_b = 20                 
if __name__ == "__main__": 
                           
a = np.arange(v)
```
when the cursor is placed just after the last `v` in `np.arange`, this is the request sent to tabnine:
```
{                                                                                                                                                                     
  request = {                                                                                                                                                         
    Autocomplete = {                                                                                                                                                  
      after = "\na = np.arange(v)",                                                                                                                                   
      before = 'var_a = 10\nvar_b = 20\nif __name__ == "__main__":\na = np.arange(v)',                                                                                
      filename = "/tmp/1.py",                                                                                                                                         
      max_num_results = 20,                                                                                                                                           
      region_includes_beginning = true,                                                                                                                               
      region_includes_end = true                                                                                                                                      
    }                                                                                                                                                                 
  },                                                                                                                                                                  
  version = "3.3.0"                                                                                                                                                   
}                      
```